### DISCUSSION TITLE
How to edit "highlightable" margin/padding color?

### DISCUSSION THREAD

**Discussion by mosh-tudor** [GrapesJS/grapesjs#3716] [Q&A] [answered]
<img width="568" alt="Screenshot 2021-08-20 at 18 56 31" src="https://user-images.githubusercontent.com/668451/130268063-21fb15c6-7bf6-42ab-9585-a9e127ad54ed.png">

How can I change these 2 colors?
Can't find them anywhere (I tried).

**Comment by artf** [comment, marked as answer]
You can simply add your custom CSS for `.gjs-margin-v-el` and `.gjs-padding-v-el`
![Schermata 2021-08-29 alle 11 50 44](https://user-images.githubusercontent.com/11614725/131246261-1da11501-876e-4ef0-b004-f42ca17b2f3e.png)
